atlasdb-api/src/main/java/com/palantir/atlasdb/transaction/api/TransactionConflictException.java

```diff
@@ -81,6 +86,29 @@ public final String toString() {
     private final ImmutableList<CellConflict> spanningWrites;
     private final ImmutableList<CellConflict> dominatingWrites;
     private final TableReference conflictingTable;
+    private final List<Arg<?>> args;
+
+    private TransactionConflictException(
+            String message,
+            Collection<CellConflict> spanningWrites,
+            Collection<CellConflict> dominatingWrites,
+            TableReference conflictingTable) {
+        super(message);
+        this.spanningWrites = ImmutableList.copyOf(spanningWrites);
+        this.dominatingWrites = ImmutableList.copyOf(dominatingWrites);
+        this.conflictingTable = conflictingTable;
+        this.args = List.of(SafeArg.of("conflictingTable", conflictingTable));
```

Comment:
table references are not always safe, `LoggingArgs::tableRef`? 

From what I can tell the existing version funnels up the message (which seems / should be unsafe) to a normal runtime exception which should have its message logged as unsafe